subprojects/signing/src/integTest/groovy/org/gradle/plugins/signing/SigningPublicationsIntegrationSpec.groovy

```diff
@@ -328,10 +328,6 @@ class SigningPublicationsIntegrationSpec extends SigningIntegrationSpec {
                 repositories {
                     ivy {
                         url "file://\$buildDir/ivyRepo/"
-                        patternLayout {
```

Comment:
❓ Why remove the `patternLayout` configuration? Aren't we losing test coverage that way?

Reply:
It's interesting and not clear for me why `patternLayout` is need to be configured in this tests?
Patterns behaviour as I see is covered by `DefaultIvyArtifactRepositoryTest`. 
But at the same time - I see that many other tests that configuring `reporsitory { ivy {} }` are applying custom `patternLayout`. I'm wondering why default pattern is not work? @bamboo Could you explain  ?

Reply:
`patternLayout` tells Gradle how the repository is organized (where each file goes), so before your change, this test was also testing that the signature files were published to the locations specified by `patternLayout`. I haven't looked if we have coverage for signature files and `patternLayout` elsewhere.

While investigating this I also found https://github.com/gradle/gradle/issues/14267 which might be the reason why you had to change these tests in the 1st place?